Added LED_0606

SchrodingersGat opened the pull request:
This PR adds the LED_0606 (1616 metric) footprint, small LED in 1.6x1.6mm package

e.g. http://www.cree.com/~/media/Files/Cree/LED-Components-and-Modules/XLamp/Data-and-Binning/ds-XQA.pdf

jkriege2 commented:
Hi!

could you add Fab-Layer? Should we exchange the K for C (like Cathode?) 

JAN

SchrodingersGat commented:
I have added fab layer. I've also removed the K as I don't think it is needed.

Maybe it's different in Germany but I always thought that K was standard for cathode...
